docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -58,7 +58,10 @@ Settings are managed using the NuGet CLI [config command](../reference/cli-refer
 Windows:
 
 ```cli
-# Set repositoryPath in the user-level config file
+# Set globalPackagesFolder (available for PackageReference only) in the user-level config file
```

Comment:
The global package folder is still used by packages.config, to avoid downloading packages from the internet. It's just also copied to the solution packages folder (repository path), which is used by the compiler and build system.

Reply:
Is this really the place to go into that level of detail? The point here is to make it clear that the command works for a variety of settings, and not all settings apply in all scenarios.

Reply:
I think "available for PackageReference only" is somewhere between completely wrong, or just misleading.  If you don't think this is the right place to explain that the global packages folder is still used as a download cache for packages.config, that's fine.  But I hope you can understand that with the way this is currently phrased that some customers would expect that packages.config packages will never be extracted to this location, which is just not true.

Reply:
So, the fact that the docs for globalPackagesFolder have told customers this is also wrong, I assume?

> globalPackagesFolder (projects using PackageReference only)

https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#config-section

Reply:
> this is also wrong, I assume?

In my opinion yes.  If you perform a restore of a packages.config project with ProcMon open, you'll see:
1. NuGet downloads the nupkg to the http-cache directory (to a temporary name then renames to the final name)
2. NuGet copies the nupkg from the http-cache directory to the global packages directory, and extracts it
3. NuGet copies the nupkg from the global packages directory to the solution packages directory and extracts it

One way to interpret "projects using PackageReference only" is that modifying the configuration file will have zero impact on projects using packages.config. However, unless what I wrote above is wrong, then the PackageReference only statement is wrong, unless the context is explained.

Do you disagree that without context, as is currently the case, that some customers might read these docs and assume that the global packages directory should remain unused by projects using packages.config, and therefore their global packages directory should be permanently empty/unused if they only ever use packages.config projects?

Reply:
Yes, clearly what you're saying PC does with GPF is not in alignment with the existing docs.

Reply:
FWIW, I just found a customer filed issue complaining about this precise documentation bug:
* https://github.com/NuGet/docs.microsoft.com-nuget/issues/2645